source/content/glossar/bibliothek/dcat-examples/dataset-landing-page-ttl.rst

```diff
@@ -1,5 +1,5 @@
 .. code-block:: Turtle
-    :caption: The landing Page in the current implementation is a string
+    :caption: The landing page in the current implementation is a string
```

Comment:
as the conformancy-aspect is removed from the handbook, I'd try to remove the word current.
`The landing page is a string.`
suffices.